MotDuJour/ComposableArchitecture/Sources/Reducers.swift

```diff
@@ -1 +1,11 @@
 public typealias Reducer<Value, Action> = (inout Value, Action) -> Void
+
+public func combine<Value, Action>(_ reducers: (inout Value, Action) -> Void...) -> (
```

Comment:
Nit: can you use the typealias here? `_ reducers: Reducer<Value, Action>...`

Reply:
yeah for sure